src/registry.ts

```diff
@@ -0,0 +1,93 @@
+import { AxiosResponse } from "axios";
+import { FileID, User } from "./skydb";
+import { defaultOptions } from "./utils";
+
+const defaultRegistryOptions = {
+  ...defaultOptions("/skynet/registry"),
+};
+
+export type RegistryValue = {
+  tweak: string;
+  data: string;
+  revision: number;
+};
+
+export type SignedRegistryValue = {
+  value: RegistryValue;
+  signature: string;
+};
+
+export async function lookupRegistry(
+  user: User,
+  fileID: FileID,
+  customOptions = {}
+): Promise<SignedRegistryValue | null> {
+  const opts = {
+    ...defaultRegistryOptions,
+    ...customOptions,
```

Comment:
the order of `customOptions` and `this.customOptions` should be reversed

Reply:
Also elsewhere

Reply:
thanks